test/gtest/uct/test_md.cc

```diff
@@ -408,12 +412,13 @@ UCS_TEST_SKIP_COND_P(test_md, alloc,
     pack_params.flags      = UCT_MD_MKEY_PACK_FLAG_INVALIDATE_RMA |
                              UCT_MD_MKEY_PACK_FLAG_INVALIDATE_AMO;
 
+    max_size               = ucs_min(max_size, 63356);
```

Comment:
Is it changed on purpose from SHRT_MAX(65536) to 63356? Maybe need a comment to explain this magic number

Reply:
It's a typo, right? @yosefe